protocol/0056-REWA-rewards_overview.md

```diff
@@ -486,10 +485,58 @@ Market goes above the threshold in trading value in an epoch before the reward a
     * Transfer 20000 USDC to `ETHUSDT | market creation | USDC` 
 
 #### Expectation
-At the end of the epoch 2 and at the end of epoch 3 no payout has been made for the market ETHUSDT and the reward account balances should remain unchanged.
+At the end of the epoch 3 a payout is made for the market ETHUSDT to the creator.
 
 
+### Distributing market creation rewards - multiple asset rewards (<a name="0056-REWA-043" href="#0056-REWA-043">0056-REWA-043</a>)
+#### Rationale 
+A market should be able to be rewarded multiple times if several reward pools are created with different payout assets.
 
+#### Setup
+* Setup a market ETHUSDT settling in USDT.
+* The value of `marketCreationQuantumMultiple` is `10^6` and `quantum` for `USDT` is `1`. 
+* Setup and fund multiple reward account for the market ETHUSDT:
+    * Transfer 10000 $VEGA to `ETHUSDT | market creation | $VEGA` 
+* start trading in the market such that traded value for fee purposes in USDT is less than 10^6
+* During the epoch 2 let the traded value be greater than 10^6
+* During epoch 3, transfer 20000 USDC to `all | market creation | USDC` 
```

Comment:
I think we need two cases here, one as you have it where the payout happens sequentially (in separate epochs), and one where the payouts takes place at the same time (i.e. setup both transfers up front, then when threshold is reached payout is made in both assets at the same time).